asset/js/setup.js

```diff
@@ -19,13 +19,52 @@ function flattenModals() {
   });
 }
 
+function insertCss(cssCode) {
+  const newNode = document.createElement('style');
+  newNode.innerHTML = cssCode;
+  document.getElementsByTagName('head')[0].appendChild(newNode);
+}
+
 function setupAnchors() {
+  const headerSelector = jQuery('header');
+  const isFixed = headerSelector.filter('.header-fixed').length !== 0;
+  const headerHeight = headerSelector.height();
+  const topPadding = 1;
+  if (isFixed) {
+    jQuery('.nav-inner').css('padding-top', `calc(${headerHeight}px + ${2 * topPadding}rem)`);
```

Comment:
It would be good if you could add some comments to explain why changing this padding is needed.